inc/class-blocks-animation.php

```diff
@@ -133,6 +106,88 @@ public function enqueue_block_frontend_assets() {
 		wp_script_add_data( 'otter-typing', 'defer', true );
 	}
 
+	/**
+	 * Load Gutenberg assets.
+	 *
+	 * @since   1.0.0
+	 * @access  public
+	 */
+	public function enqueue_block_frontend_assets() {
+		if ( function_exists( 'is_amp_endpoint' ) && is_amp_endpoint() ) {
+			self::$can_load_frontend = false;
+		}
+
+		if ( is_singular() && strpos( get_the_content( null, false, $post ), '<!-- wp:' ) === false ) {
```

Comment:
That error still appears if I preview or view a post. This happens because $post isn't defined anywhere. @Soare-Robert-Daniel